packages/tables/src/Columns/Concerns/CanUpdateState.php

```diff
@@ -30,6 +32,8 @@ public function updateState(mixed $state): mixed
             ]);
         }
```

Comment:
IMO, the hooks should still be run, even if you're using `updateStateUsing()`

Reply:
done ✅